engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/query/NadelFieldToService.kt

```diff
@@ -0,0 +1,54 @@
+package graphql.nadel.enginekt.transform.query
+
+import graphql.nadel.Service
+import graphql.nadel.enginekt.blueprint.NadelOverallExecutionBlueprint
+import graphql.nadel.enginekt.util.copyWithChildren
+import graphql.nadel.schema.NadelDirectives
+import graphql.normalized.NormalizedField
+import graphql.normalized.NormalizedQuery
+import graphql.schema.GraphQLSchema
+
+class NadelFieldToService {
+
+    fun getServicesForTopLevelFields(query: NormalizedQuery, overallExecutionBlueprint: NadelOverallExecutionBlueprint): List<FieldAndService> {
+        return query.topLevelFields.flatMap { topLevelField ->
+            when {
+                NadelNamespacedFields.isNamespacedField(topLevelField, overallExecutionBlueprint.schema) -> {
+                    val namespacedChildFieldsByService: Map<Service, List<NormalizedField>> =
+                        topLevelField.children
+                            .groupBy {
```

Comment:
I'm thinking now… this `groupBy` changes the behavior a lot compared to our normal execution.

Say we get a query:

```graphql
{
  jira {
    issues
    projects
  }
}
```

If these were normal top level fields like:

```
{
  issues
  projects
}
```

Then we would execute the top level fields separately and then merge them. But here because of the `groupBy` we send the two fields down together.

cc: @andimarek thoughts on this difference in behavior? I think we should retain the old behavior since it's just a namespacing.